agr_literature_service/api/crud/reference_crud.py

```diff
@@ -230,6 +223,28 @@ def show_all_references_external_ids(db: Session):
             for reference in references_query.all()]
 
 
+def get_merged(db: Session, curie):
+    reference = None
+    logger.debug("Looking up if '{}' is a merged entry".format(curie))
+    # Is the curie in the merged set
+    try:
+        orc: ObsoleteReferenceModel = db.query(ObsoleteReferenceModel).filter(ObsoleteReferenceModel.curie == curie).one_or_none()
```

Comment:
orc == obsolete reference curie ?  (just checking that it's not related to orcid

Reply:
correct. I might lengthen it to make it clearer.

Reply:
cool, thanks for clarifying